releasenotes/notes/prepare-2.0.0-bab067ae93d40bb1.yaml

```diff
@@ -0,0 +1,25 @@
+---
+prelude: >
+    We are pleased to release Qiskit 2.0.0, with 
+    new features that improve performance and expand Qiskit's capabilities.
+    In line with our semantic versioning policy, documented a the time of 
+    the 1.0.0 release, this major release also includes API changes that 
+    are not backward-compatible with the 1.x series. 
+    
+    The feature highlights of Qiskit 2.0.0 are:
```

Comment:
I would also suggest to add:
- circuit library new functions
- fractional gates synthesis support https://www.ibm.com/quantum/blog/fractional-gates

Reply:
I think fractional gate is a good one. however, "new circuit library new functions" seems to me too general. What are those new functions?

Reply:
I meant the re-org of the circuit library: `quantum_volume` instead of `QuantumVolume` etc.

Reply:
thanks Shelly, the circuit library one was already announced in 1.3, but I can add the fractional gates. I will probably have to reword the part about performance too.